File: spaceStation/src/main/java/challenge/spacestation/data/modelos/Tripulada.java
package challenge.spacestation.data.modelos;

/**
 *
 * @author mcami
 */
public class Tripulada extends Nave{
    public int capacity;
    public String objective;

    public Tripulada() {
    }

    public Tripulada(int capacity, String objective, int id, String name, String country, int activity_start, int activity_end, String fuel) {
        super(id, name, country, activity_start, activity_end, fuel);
        this.capacity = capacity;
        this.objective = objective;
    }

    public int getCapacity() {
        return capacity;
    }

    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }

    public String getObjective() {
        return objective;
    }

    public void setObjective(String objective) {
        this.objective = objective;
    }
}

File: app/database/models/Tripulada.js
module.exports = (sequelize, dataTypes) => {
    let alias = 'Tripulada';
    let cols = {
        id: {
            type: dataTypes.INTEGER.UNSIGNED,
            primaryKey: true,
            autoIncrement: true,
            allowNull: false
        },
        name: {
            type: dataTypes.STRING(50),
            allowNull: false
        },
        country: {
            type: dataTypes.STRING(50)
        },
        activity_start: {
            type: dataTypes.INTEGER
        },
        activity_end: {
            type: dataTypes.INTEGER
        },
        fuel: {
            type: dataTypes.STRING(50)
        },
        capacity: {
            type: dataTypes.INTEGER
        },
        objective: {
            type: dataTypes.STRING(50)
        }
    };
    let config = {
        tableName: 'naves_tripuladas',
        timestamps: false
    }
    const Tripulada = sequelize.define(alias, cols, config); 
 
    return Tripulada
};